doc/api/process.md

```diff
@@ -1684,6 +1684,107 @@ the [`'warning'` event][process_warning] and the
 [`emitWarning()` method][process_emit_warning] for more information about this
 flag's behavior.
 
+## `process.parseArgs([argv[, options]])`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `argv` {Array<string>|Object} (Optional) Array of argument strings; defaults
+  to [`process.argv.slice(2)`](process_argv). If an Object, the default is used,
+  and this parameter is considered to be the `options` parameter.
+* `options` {Object} (Optional) The `options` parameter, if present, is an
+  object supporting the following property:
+  * `expectsValue` {Array<string>|string} (Optional) One or more argument
```

Comment:
We should rename this to something else that:

- Does not contain "expect" in the name (when I see an option with `expects*`, I expect it to validate the existence of something, in other words, I would expect this to make those options required).
- Contains the word 'option'. I _think_ the intention here was "options that expect values", but the lack of the word 'option' makes it less intuitive

What about `optionsWithValues` or `optionsAcceptingValues`?

Reply:
I like `optionsWithValues`.